services/webhook/webhook.go

```diff
@@ -179,17 +180,32 @@ func prepareWebhook(w *models.Webhook, repo *models.Repository, event models.Hoo
 		signature = hex.EncodeToString(sig.Sum(nil))
 	}
 
+	var signaturegithub string
+	if len(w.Secret) > 0 {
+		data, err := payloader.JSONPayload()
+		if err != nil {
+			log.Error("prepareWebhooks.JSONPayload: %v", err)
+		}
+		sig := hmac.New(sha1.New, []byte(w.Secret))
+		_, err = sig.Write(data)
+		if err != nil {
+			log.Error("prepareWebhooks.sigWrite: %v", err)
+		}
+		signaturegithub = "sha1=" + hex.EncodeToString(sig.Sum(nil))
```

Comment:
@techknowlogick thanks for the quick reply. This is my first PR and I might be missing some obvious things for sure :)

current signature is in **sha256** format and X-Hub-Signature expects **sha1**, at least that seems to be the standard for GitHub. I tried using sha256= in the header but that doesn't seem to be supported at least with the upstream tool I used. It could be a bug on the other tool which may be hardcoded to read sha1 always.

I also tried using X-Hub-Signature-256 so that I could directly pass the existing signature but also didn't work for me.

Current signature is created with:
`sig := hmac.New(sha256.New, []byte(w.Secret))`
I create the additional signature with:
`sig := hmac.New(sha1.New, []byte(w.Secret))`